jdi-light-mobile-tests/src/main/java/pseudo/site/forms/CustomForm.java

```diff
@@ -0,0 +1,284 @@
+package pseudo.site.forms;
+
+import com.epam.jdi.light.elements.common.UIElement;
+import com.epam.jdi.light.elements.complex.WebList;
+import com.epam.jdi.light.elements.complex.dropdown.Dropdown;
+import com.epam.jdi.light.elements.composite.Form;
+import com.epam.jdi.light.elements.pageobjects.annotations.FindBy;
+import com.epam.jdi.light.elements.pageobjects.annotations.locators.*;
+import com.epam.jdi.light.ui.html.elements.common.Button;
+import org.openqa.selenium.WebElement;
+import pseudo.site.entities.User;
+
+import java.util.List;
+
+public class CustomForm extends Form<User> {
+    @FindBy(css = "#fbWebElement-id") WebElement fbWebElement;
+    @FindBy(css = "#fbPublicWebElement-id") public WebElement fbPublicWebElement;
+    @FindBy(css = "#fbPrivateWebElement-id") private WebElement fbPrivateWebElement;
+    @UI("#uiWebElement-id") WebElement uiWebElement;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use explicit scoping instead of the default package private level](https://app.codacy.com/manual/42_/jdi-light/pullRequest?prid=5217753)